src/makeRobot.js

```diff
@@ -38,7 +38,69 @@
  * @return {object}
  */
 function makeRobot(name, wheels, version) {
-  // write code here
+  const robot = {
+    name: name,
+    version: version,
+    whells: wheels,
+    coords: {
+      x: 0,
+      y: 0,
+    },
+    get info() {
+      const Name = this.name;
+      const Version = this.version;
+      const Whells = this.whells;
+
+      return `name: ${Name}, chip version: ${Version}, wheels: ${Whells}`;
+    },
+    get location() {
+      return `${this.name}: x=${this.coords.x}, y=${this.coords.y}`;
+    },
+    goForward(value = 1) {
+      if (value < 0) {
+        return this;
+      };
+
+      this.coords.y += value;
+
+      return this;
+    },
+    goBack(value = 1) {
+      if (value < 0) {
+        return this;
+      };
+
+      this.coords.y -= value;
+
+      return this;
+    },
+    goLeft(value = 1) {
+      if (value < 0) {
+        return this;
+      };
+
+      this.coords.x -= value;
+
+      return this;
+    },
+    goRight(value = 1) {
+      if (value < 0) {
+        return this;
+      };
+
+      this.coords.x += value;
+
+      return this;
+    },
+    evacuate() {
+      this.coords.x = 1400;
+      this.coords.y = 500;
+
+      return this;
+    },
```

Comment:
no need to return this in evacuate

```suggestion
    evacuate() {
      this.coords.x = 1400;
      this.coords.y = 500;
    },
```